apps/lend/src/layout/Header.tsx

```diff
@@ -117,7 +117,15 @@ const Header = () => {
   const handleNetworkChange = (selectedChainId: React.Key) => {
     if (rChainId !== selectedChainId) {
       const network = networks[selectedChainId as ChainId].id
-      navigate(`${rLocalePathname}/${network}/${getRestPartialPathname()}`)
+      const [currPath] = window.location.hash.split('?')
+
+      if (currPath.endsWith('markets')) {
```

Comment:
It is generally useful to keep page URLs in constants so they can be updated without introducing bugs